packages/design-tokens/src/css/typography.css

```diff
@@ -3,9 +3,9 @@
  */
 :root {
   /* font family */
-  --font-family-euclid-circular-b: 'Euclid Circular B', sans-serif;
-  --font-family-roboto: 'Roboto', sans-serif;
-  --font-family-sans: 'Euclid Circular B', 'Roboto', sans-serif;
+  --font-family-centra-no1: 'Centra No1', 'Helvetica Neue', Helvetica, Arial,
+    sans-serif;
+  --font-family-sans: var(--font-family-centra-no1);
```

Comment:
Added *CentraNo1* and removed *Euclid* and *Roboto* CSS variables. Since switching the theme font family token to *CentraNo1* is a breaking change, we’re also removing the old variables altogether.